### ISSUE TITLE
URL.createObjectURL with media streams is deprecated and will be removed in Google Chrome soon

### ISSUE THREAD

**Issue by He4eT** [Kurento/bugtracker#237]
```
kurento-utils.js:292 [Deprecation] URL.createObjectURL with media streams is deprecated and will be removed in M68, around July 2018. Please use HTMLMediaElement.srcObject instead. See https://www.chromestatus.com/features/5618491470118912 for more details.
setRemoteVideo @ kurento-utils.js:292
(anonymous) @ kurento-utils.js:326
(anonymous) @ adapter.js:361
```

**Comment by ruddell** [comment]
This is already fixed in v6.7.0 of Kurento Utils https://github.com/Kurento/kurento-utils-js/commit/a7bdf3c58518cb595f9b5b9fd28d8392cc303c05

**Comment by He4eT** [comment]
Thanx!
